pkg/providers/gcp/cluster_network_provider.go

```diff
@@ -24,10 +29,15 @@ const (
 	defaultIpRangePostfix          = "ip-range"
 	defaultIpRangeCIDRMask         = 22
 	defaultNumberOfExpectedSubnets = 2
+	defaultServicesFormat          = "services/%s"
+	defaultServiceConnectionFormat = defaultServicesFormat + "/connections/%s"
+	defaultNetworksFormat          = "projects/%s/global/networks/%s"
+	defaultIpv4Length              = 8 * net.IPv4len
 )
 
+//go:generate moq -out cluster_network_provider_moq.go . NetworkManager
 type NetworkManager interface {
```

Comment:
I do agree on this but I'm still a little unsure - I think we've moved implementation for each interface to `gcpiface` too which would result in all but the helper functions being moved across so I think there wouldn't be much left in the network_cluster_provider. Perhaps we just leave it as is so that this file sort of matches aws?